code/game/machinery/newscaster/newscaster_machine.dm

```diff
@@ -0,0 +1,551 @@
+				tgui_alert(usr, "ERROR: User cannot be found or already has an owned feed channel.", list("Okay"))
+				return
+			creating_channel = TRUE
+		if("channelName")
+			channel_name = (params["channeltext"])
+		if("channelDesc")
+			channel_desc = (params["channeldesc"])
+
+		if("createChannel")
+			if(!channel_name)
+				return
+			for(var/datum/newscaster/feed_channel/FeedC in GLOB.news_network.network_channels)
+				if(FeedC.channel_name == channel_name)
+					tgui_alert(usr, "ERROR: Feed Channel with that name already exists on the Network.", list("Okay"))
+					return
+			if(!channel_desc)
+				return
+			var/locked = (params["publicmode"])
+			var/choice = tgui_alert(usr, "Please confirm Feed channel creation","Network Channel Handler", list("Confirm","Cancel"))
+			if(choice=="Confirm")
+				GLOB.news_network.CreateFeedChannel(channel_name, current_user.account_holder, channel_desc , locked)
+				SSblackbox.record_feedback("text", "newscaster_channels", 1, "[channel_name]")
+			creating_channel = FALSE
+
+		if("storyCensor")
+			if (!params["secure"])
+				say("Clearance not found.")
+				return
+			var/questionable_message = params["messageID"]
+			for(var/datum/newscaster/feed_message/mess in current_channel.messages)
+				if(mess.message_ID == questionable_message)
+					mess.toggleCensorBody()
+					break
+
+		if("authorCensor")
+			if (!params["secure"])
+				say("Clearance not found.")
+				return
+			var/questionable_message = params["messageID"]
+			for(var/datum/newscaster/feed_message/mess in current_channel.messages)
+				if(mess.message_ID == questionable_message)
+					mess.toggleCensorAuthor()
+					break
+
+		if("channelDNotice")
+			if (!params["secure"])
+				say("Clearance not found.")
+				return
+			var/proto_chan = (params["channel"])
+			for(var/datum/newscaster/feed_channel/potential_channel in GLOB.news_network.network_channels)
+				if(proto_chan == potential_channel.channel_ID)
+					current_channel = potential_channel
+					break
+			current_channel.toggleCensorDclass()
+
+		if("printNewspaper")
+			if(paper_remaining <= 0)
+				balloon_alert_to_viewers("Unit out of Paper!")
+				return
+			print_paper()
+
+		//**************************
+		//	  Bounty Board Acts
+		//**************************
+		if("createBounty")
+			if(!current_user || !bounty_text)
+				playsound(src, 'sound/machines/buzz-sigh.ogg', 20, TRUE)
+				return TRUE
+			for(var/datum/station_request/i in GLOB.request_list)
+				if("[i.req_number]" == "[current_user.account_id]")
+					say("Account already has active bounty.")
+					return
+			var/datum/station_request/curr_request = new /datum/station_request(current_user.account_holder, bounty_value,bounty_text,current_user.account_id, current_user)
+			GLOB.request_list += list(curr_request)
+			for(var/obj/i in GLOB.allbountyboards)
+				balloon_alert_to_viewers("New bounty has been added!")
```

Comment:
You're not using `i` at all here, this is just going to display a billion balloon alerts on this one board.

Also, this should be `say`.